samples/app/src/androidTest/java/com/sourcepoint/example_app/ExampleAppTests.kt

```diff
@@ -0,0 +1,197 @@
+package com.sourcepoint.example_app
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.launchActivity
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.example.uitestutil.wr
+import com.sourcepoint.example_app.TestUseCase.Companion.checkConsentAsSelectedFromPartialConsentList
+import com.sourcepoint.example_app.TestUseCase.Companion.checkConsentIsNotSelected
+import com.sourcepoint.example_app.TestUseCase.Companion.checkConsentIsSelected
+import com.sourcepoint.example_app.TestUseCase.Companion.checkMainWebViewDisplayed
+import com.sourcepoint.example_app.TestUseCase.Companion.checkPMTabSelectedFeatures
+import com.sourcepoint.example_app.TestUseCase.Companion.checkPMTabSelectedPurposes
+import com.sourcepoint.example_app.TestUseCase.Companion.checkPartialConsentIsSelected
+import com.sourcepoint.example_app.TestUseCase.Companion.checkWebViewDisplayedForMessage
+import com.sourcepoint.example_app.TestUseCase.Companion.checkWebViewDisplayedForPrivacyManager
+import com.sourcepoint.example_app.TestUseCase.Companion.clickOnReviewConsent
+import com.sourcepoint.example_app.TestUseCase.Companion.clickPMTabSelectedPurposes
+import com.sourcepoint.example_app.TestUseCase.Companion.setFocusOnLayoutActivity
+import com.sourcepoint.example_app.TestUseCase.Companion.tapAcceptAllOnWebView
+import com.sourcepoint.example_app.TestUseCase.Companion.tapAcceptOnWebView
+import com.sourcepoint.example_app.TestUseCase.Companion.tapDismissWebView
+import com.sourcepoint.example_app.TestUseCase.Companion.tapOptionWebView
+import com.sourcepoint.example_app.TestUseCase.Companion.tapRejectAllWebView
+import com.sourcepoint.example_app.TestUseCase.Companion.tapRejectOnWebView
+import com.sourcepoint.example_app.TestUseCase.Companion.tapSaveAndExitWebView
+import kotlinx.coroutines.runBlocking
+import org.junit.After
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+class ExampleAppTests {
+
+    lateinit var scenario: ActivityScenario<MainActivity>
+
+    @After
+    fun cleanup() {
+        if(this::scenario.isLateinit) scenario.close()
+    }
+
+    private val d = 1000L
+
+    @Test
+    fun checkAcceptActionFromDirectPrivacyManager() = runBlocking<Unit> {
+
+        scenario = launchActivity()
+
+        wr(d) { checkWebViewDisplayedForMessage() } // Assert.assertTrue(checkWebViewDisplayedFor(MESSAGE));
```

Comment:
@carmelo-iriti shall we remove the commented out code? Or is it something we would like to keep for later?

Reply:
It would be good to remove them after we rewrite all the missing tests